core/bentley/package.json

```diff
@@ -33,6 +33,8 @@
     "url": "http://www.bentley.com"
   },
   "devDependencies": {
+    "@opentelemetry/api": "^1.0.4",
```

Comment:
Does this not need to be a real dependency to use the `tracer` class?

Reply:
true, I accidentally put it in the wrong section

Reply:
If the openTelemetry is going to be optional, not sure if we should be adding this as a dependency for all apps...  Especially since it's mostly going to be used in the web scenario.

Reply:
hm, I might have a way to move it to `logger-config`. Marking this as draft for now.

Reply:
I made this an optional peerDependency of core-backend. Now it's enough to just have `@opentelemetry/api` in your dependencies and core-backend will pick it up and enable tracing. If you don't have it installed, current behavior won't change.